tests/Behat/features/broken-element.feature

```diff
@@ -0,0 +1,37 @@
+@javascript
+Feature: Broken element in CMS
+  As a CMS user
+  I want elemental areas to function correctly even with a broken block
+  So that I can continue to update my content
+
+
+  Background:
+    Given I add an extension "DNADesign\Elemental\Extensions\ElementalPageExtension" to the "Page" class
+      And a "page" "Blocks Page" with a "Alice's Block" content element with "Some content" content
+      And a "page" "Blocks Page" has a broken element named "This one is broken"
+      And the "page" "Blocks Page" has a "Bob's Block" content element with "Some content II" content
+
+    Given I am logged in with "ADMIN" permissions
+      And I go to "/admin/pages"
+      And I left click on "Blocks Page" in the tree
+    When I see a list of blocks
+      Then I should see "Alice's Block"
+      And I should see "Bob's Block"
+      And I should see "This element is of obsolete type BrokenClass"
+      And I should see "This one is broken"
+
+  @modal
+  Scenario: I can archive a block
+    Given I see a list of blocks
+    When I press the "View actions" button
+      Then I should see the archive button for block 2
```

Comment:
Small question. Should we check that we don't see "Save" and "Publish" actions?

Reply:
Probably worth checking for that, yeah. Will add that.

Reply:
Done. Also checking for duplicate button for completeness.
Adding this required some small refactoring of the behat `FeatureContext` - all the same behat test scenarios still work but now there is no duplication of code, and we can check for any action button in a uniform way instead of having a separate method for each possible action.